useful workflows/target_workflow_2/target_workflow_2-test.wdl

```diff
@@ -0,0 +1,413 @@
+  }
+
+  command {
+    echo "lofreq_mock"
+  }
+
+  output {
+    File   output_file   = input_file_1
+    String output_string = input_string_default_1
+  }
+}
+
+task alignment_metrics_mock {
+  meta {}
+
+  input {
+    File   input_file_1
+    File   input_file_2
+    File?  input_file_optional
+    String? input_string_optional
+    Int?   input_int_optional_1
+    Int?   input_int_optional_2
+    Int?   input_int_optional_3
+
+    Int?   input_int_optional_4
+    String docker = "quay.io/broadinstitute/viral-core:2.1.33"
+  }
+
+  command {
+    echo "alignment_metrics_mock"
+  }
+
+  output {
+    File output_file_1 = input_file_1
+    File output_file_2 = input_file_2
+    File output_file_3 = input_file_1
+    File output_file_4 = input_file_2
+    File output_file_5 = input_file_1
+  }
+}
+
+task run_discordance_mock {
+  input {
+    File   input_file_1
+    File   input_file_2
+    String input_string_default = "run"
+    Int    input_int_default = 4
+
+    String docker = "quay.io/broadinstitute/viral-core:2.1.33"
+  }
+  
+  command {
+    echo "run_discordance_mock"
+  }
+  
+  output {
+    File   output_file   = input_file_1
+    Int    output_int_1  = input_int_default
+    Int    output_int_2  = input_int_default
+    Int    output_int_3  = input_int_default
+    Int    output_int_4  = input_int_default
+    Int    output_int_5  = input_int_default
+    String output_string = input_int_default
+  }
+}
+
+task plot_coverage_mock {
+  input {
+    File    input_file
+    String  input_string
+
+    Boolean input_bool_default_1  = false
+    Boolean input_bool_default_2  = false
+    Boolean input_bool_default_3  = false
+    String? input_string_default  = "max" # max or min
+
+    Int? input_int_default_1      = 1100
+    Int? input_int_default_2      = 850
+    Int? input_int_default_3      = 100
+
+    Int? input_int_optional_1
+    Int? input_int_optional_2
+    Int? input_int_optional_3
+    Int? input_int_optional_4
+    String? input_string_optional_1 # of the form "min max" (ints, space between)
+    String? input_string_optional_2 # of the form "min max" (ints, space between)
+
+    String docker = "quay.io/broadinstitute/viral-core:2.1.33"
+  }
+
+  command {
+    echo "plot_coverage_mock"
+  }
+
+  output {
+    File   output_file_1  = input_file
+    File   output_file_2  = input_file
+    Int    output_int_1   = 1
+    Int    output_int_2   = 2
+    Int    output_int_3   = 3
+    Float  output_float_1 = 4.0
+    Float  output_float_2 = 5.0
+    String output_string  = "hello world"
+  }
+}
+
+task refine_assembly_with_aligned_reads_mock {
+
+  input {
+    File     input_file_1
+    File     input_file_2
+    String   input_string
+
+    Boolean  input_boolean_default = false
+    Float    input_float_default = 0.5
+    Int      input_int_default = 3
+
+    Int?     input_int_optional
+    String   docker = "quay.io/broadinstitute/viral-assemble:2.1.16.1"
```

Comment:
I was about to say "let's use tiny docker images rather than full ones... but use of `quay` might be a nice test case once this hits the "can Cromwell run it" stage...

Reply:
interestingly, this was leftover, and isn't being used anywhere. I've deleted all the `runtime` blocks. Is that a mistake? Should I put them back in?

Reply:
Yes, we'll need to have at least a docker image declared in every task for it to be able to run in CaaA